app/src/main/java/org/p2p/wallet/deeplinks/AppDeeplinksManager.kt

```diff
@@ -0,0 +1,76 @@
+package org.p2p.wallet.deeplinks
+
+import android.app.ActivityManager
+import android.content.Context
+import android.content.Intent
+import androidx.core.content.getSystemService
+import org.p2p.wallet.R
+import org.p2p.wallet.notification.NotificationType
+import org.p2p.wallet.root.RootActivity
+import org.p2p.wallet.utils.toStringMap
+
+class AppDeeplinksManager(private val context: Context) {
+
+    companion object {
+        const val NOTIFICATION_TYPE = "eventType"
+        const val DEEPLINK_MAIN_SCREEN_EXTRA = "DEEPLINK_SCREEN_EXTRA"
+    }
+
+    var mainTabsSwitcher: MainTabsSwitcher? = null
+
+    private var pendingIntent: Intent? = null
+
+    fun buildIntent(notificationType: NotificationType): Intent {
+        val activityManager = context.getSystemService<ActivityManager>()
+        val intent = activityManager?.appTasks
```

Comment:
which Intent is this?
I suggest naming it
```suggestion
        val openedScreenOrRoot = activityManager?.appTasks
```

Reply:
okay =)